### DISCUSSION TITLE
Is the Twitter shortcode working?

### DISCUSSION THREAD

**Discussion by betolink** [adityatelange/hugo-PaperMod#1179] [Q&A] [answered]
I tried to embed a tweet using PaperMod and when I rendered the site I got the following error 

```
hugo v0.111.3-5d4eb5154e1fed125ca8e9b5a0315c4180dab192+extended linux/amd64 BuildDate=2023-03-12T11:40:50Z VendorInfo=snap:0.111.3
ERROR 2023/03/26 11:36:00 Failed to get JSON resource "https://publish.twitter.com/oembed?dnt=false&url=https%3A%2F%2Ftwitter.com%2F%E2%80%9Cdustybowl%E2%80%9D%2Fstatus%2F%E2%80%9C1504481406122008588%E2%80%9D": Failed to retrieve remote file: Not Found,
```

I'm not sure if this is a misconfiguration issue or there is a bug with Twitter's partial template.
My guess is that Twitter API changed recently because if we try to embed a tweet now it gives us a different URL:

`https://publish.twitter.com/?query=` instead of `https://publish.twitter.com/oembed?`

**Comment by chrisrummel** [comment, marked as answer]
This isn't working since at least 2022-11-18, shortly after Musk took over.
